src/Relude/Extra/Enum.hs

```diff
@@ -58,7 +59,7 @@ inverseMap :: forall a k . (Bounded a, Enum a, Ord k)
 inverseMap f = \k -> M.lookup k dict
   where
     dict :: M.Map k a
-    dict = M.fromList $ zip (map f univ) univ
+    dict = M.fromList $ map (mapToFst f) univ
```

Comment:
Yep, using `map` instead of `zip` here was in my plans for this function. While we are on this, let's refactor some more and have the following defintion of `dict`:

```haskell
dict = M.fromList $ map (mapToFst f) (universe @a)
```

No need to have an extra variable, we can make a function 3 lines shorter and more understandable.

Reply:
Done.